index.html

```diff
@@ -237,16 +237,13 @@ <h2>Scope</h2>
 						Such new technical features should be incubated by the <a href="https://www.w3.org/community/publishingcg/">Publishing Community Group</a> before being considered by this Working Group.  Also, the effects on deployed systems and checking tools, and the risk of additional cost for adoption, must be carefully analyzed. In case of doubt, such features should not be added to EPUB 3.X.
 					</li>
 					<li>
-						Improvements of the <a href='#a11y'>EPUB Accessibility</a> specification; these may include, but not limited to:
+						Improvements to the <a href='#a11y'>EPUB Accessibility</a> specification, that may include, but not be limited to:
 						<ul>
 							<li>
-								verify that EPUB Accessibility 1.X aligns with the <a href='https://eur-lex.europa.eu/legal-content/EN/TXT/?uri=uriserv:OJ.L_.2019.151.01.0070.01.ENG&toc=OJ:L:2019:151:TOC'>EU Directive for accessibility requirements for eBooks</a> (European Accessibility Act, Directive 2019/882 of the European Parliament) to the extent possible; the WG will conduct a review of the <a href='#a11y'>EPUB Accessibility</a> specification to ensure that it aligns with the EU accessible publishing requirements to the extent that these improve accessibility;
+								considering early adoption of EPUB Accessibility developments in Europe, while seeking a stronger international consensus through W3C discussion;
 							</li>
 							<li>
-								consider new and incremental features on accessibility, provided that the aforementioned EU accessibility requirements and the underlying WCAG 2.X requirements are not compromised; the features under consideration include accessibility requirements for fixed layout documents (FXL), or the possibility of referring to different versions of WCAG 2.X;
-							</li>
```

Comment:
"enhancing EPUB Accessibility features, for instance by: aligning with advancements in WCAG; adding accessibility requirements for fixed layout documents (FXL); updating EPUB Accessibility to align with the evolving EPUB3 suite."

In this list the highest  risk feature is adding accessibility requirements for fixed layout documents (FXL), this is why I had phrased it in a way that we will explore if it can be added. better to state:
enhancing EPUB Accessibility features, for instance by: aligning with advancements in WCAG; updating EPUB Accessibility to align with the evolving EPUB3 suite; explore possibility of adding accessibility requirements for fixed layout documents (FXL).

Reply:
I am fine with this proposed change, but I leave this to Philippe who has the pen at this moment.

Reply:
Just flagging @plehegar through his github nickname rather than his real one:-)

Reply:
it's fine adding "explore to"

Reply:
https://github.com/w3c/epub-3-wg-charter/pull/58